src/dask_awkward/layers/layers.py

```diff
@@ -86,13 +93,26 @@ def mock(self) -> AwkwardArray:
         assert self._meta is not None
         return self._meta
 
+    def mock_empty(self, backend: BackendT = "cpu") -> AwkwardArray:
+        import awkward as ak
+
+        if backend not in ("cpu", "jax", "cuda"):
```

Comment:
FWIW, despite this being a bit of an abuse of a private interface you could make a set out of the keys of https://github.com/scikit-hep/awkward/blob/main/src/awkward/_backends/dispatch.py#L24 
and substract "typetracer" from the set and use that here. Should remove and problems coming from more backends over time.

@agoose77 may have issue with this though given it being hacky.